testing/lzop/APKBUILD

```diff
@@ -0,0 +1,41 @@
+# Contributor: Sasha Gerrand <alpine-pkgs@sgerrand.com>
+# Maintainer: Sasha Gerrand <alpine-pkgs@sgerrand.com>
+pkgname=lzop
+pkgver=1.04
+pkgrel=0
+pkgdesc="lzop is a very fast file compressor"
+url="https://www.lzop.org/"
+arch="all"
+license="GPL2"
```

Comment:
this is ambiguous and not SPDX compliant

Reply:
```suggestion
license="GPL-2.0-only"
```